src/panoptes/pocs/scheduler/observation/bias.py

```diff
@@ -0,0 +1,27 @@
+import os
+from astropy import units as u
+
+from panoptes.utils.config.client import get_config
+
+from panoptes.pocs.scheduler.field import Field
+from panoptes.pocs.scheduler.observation.observation import Observation
+
+
+class BiasObservation(Observation):
+
+    def __init__(self, position, min_nexp=None, exp_set_size=None):
+        if min_nexp is None:
+            min_nexp = get_config("calibs.bias.min_nexp", default=10)
```

Comment:
Everything that pulls from `PanBase` has a `get_config`.

```suggestion
            min_nexp = self.get_config("calibs.bias.min_nexp", default=10)
```